### PR TITLE
Users dog index

### PR DESCRIPTION
**Pull request by delaneymiranda1**
## Description

This PR completes the users dogs index story. Dogs are displayed on a users dogs index page in boxes with each dogs attributes and a button to edit each that redirects to the dog edit path. The add dog button is also functioning and redirecting to the add dog path.  

## Type of change
Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe testing implemented and what all it covers. 

- [x]  fully tests happy and sad paths

## Checklist

- [x] code has been self reviewed
- [x] code runs without any errors
- [x] all tests pass

### CODE REVIEW

File: app/facades/dogs_facade.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      {
+        status: response.status,
+        dog_id: response_body[:data][:id]
+      }
+    else
+      {
+        status: response.status,
+        error: response_body[:error]
+      }
+    end
```
Comment (ttakahashi1591) [inline app/facades/dogs_facade.rb:24]: This conditional looks great here and I like the addition of the sad pathing!

File: app/services/dogs_service.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
+class DogsService
+  def self.conn
+    Faraday.new(url: "http://localhost:3000/")
+  end
+  
+  def self.get_url(url)
+    response = conn.get(url)
+    JSON.parse(response.body, symbolize_names: true)
+  end
```
Comment (ttakahashi1591) [inline app/services/dogs_service.rb:9]: Nice Job with the way you built out the service and broke out the methods when reaching out to the endpoints.

File: app/views/dogs/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+      margin-top: 20px; 
+    }
+
+    .dog-box {
+      border: 5px solid #ddd;
+      padding: 10px;
+      text-align: left;
+      width: 200px; 
+    }
+  </style>
```
Comment (ttakahashi1591) [inline app/views/dogs/index.html.erb:29]: I like the use of styling here to build out container blocks for each dog!

**Review by ttakahashi1591 — APPROVED** [review]
This all looks great, thank you for building this out and for the great communication with BE to ensure you had all the information you needed to get this implemented.

**Review by EricBelongea — APPROVED** [review]
Great work and good communication.

**Pull request merged**